Library/Formula/c.rb

```diff
@@ -0,0 +1,16 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.06.tar.gz"
+  sha1 "0eefb3523abccea7ebdd2c56d4d7be6f23585645"
+
+  depends_on "gnu-sed"
+
+  def install
+    inreplace "c", "sed", "gsed"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!